ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -78,6 +78,10 @@ def Minimum(*, x, y, name="Minimum"):
     return ivy.minimum(x, y)
 
 
+def Sub(x, y, name="Sub"):
```

Comment:
You missed a `*` in the parameters to indicate that they're keyword args :sweat_smile:

Reply:
committed in [e12679c](https://github.com/unifyai/ivy/pull/3668/commits/e12679c67aaef3661761d06cb3489dee721303e3)